tests/system-broker/tests/apitests/auth_test.go

```diff
@@ -0,0 +1,147 @@
+package apitests
```

Comment:
Ditch the `apistests` directory and place the tests directly under `tests/system-broker/tests`. There's no point for it and after refactoring the e2e tests it would be removed anyway